server/store.go

```diff
@@ -140,25 +140,25 @@ func (p *Plugin) storeMeetingPostID(meetingID int, postID string) *model.AppErro
 	return p.API.KVSetWithExpiry(key, bytes, meetingPostIDTTL)
 }
 
-func (p *Plugin) fetchMeetingPostID(meetingID string) (string, *model.AppError) {
+func (p *Plugin) fetchMeetingPostID(meetingID string) (string, error) {
 	key := fmt.Sprintf("%v%v", postMeetingKey, meetingID)
-	postID, appErr := p.API.KVGet(key)
-	if appErr != nil {
-		p.API.LogDebug("Could not get meeting post from KVStore", "error", appErr.Error())
-		return "", appErr
+	var postID string
+	if err := p.client.KV.Get(key, postID); err != nil {
+		p.client.Log.Debug("Could not get meeting post from KVStore", "error", err.Error())
```

Comment:
This is not necessarily an error we want to log right? @hanzei If a meeting was created outside of Mattermost, and then it ends, we would always log this. I'm thinking this could be excessive since this sounds like an error but it's really a regular occurrence. Or does this always mean that there was an actual error reading from the kvstore?

Reply:
@mickmister I think the KV store does not return an error if the post id is not found. It just returns an empty post ID